notebooks/minicpm-o-omnimodal-chatbot/minicpm-o-omnimodal-chatbot.ipynb

```diff
@@ -0,0 +1,808 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "torch==2.3.1" "torchvision==0.18.1" "librosa==0.9.0" "vocos==0.1.0" \
</pre><p>is is required to use torch 2.3.1 only?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2851/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Its suggested by model provider
https://huggingface.co/openbmb/MiniCPM-o-2_6#usage

Reply:
done